.github/workflows/ci.yaml

```diff
@@ -0,0 +1,117 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'main'
+
+env:
+    SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+jobs:
+    composer-validate:
+        name: "Composer validate (${{ matrix.php-version }})"
+
+        runs-on: ubuntu-16.04
```

Comment:
why using an old ubuntu environment ?

Reply:
When I migrate some things I want to use nearly the same config like on travis to ensure an error is not based on the OS version.
I updated to `ubuntu-latest` now